### PR TITLE
Copied

### PR DESCRIPTION
**Pull request by Kaneez98Fatima**
Create a new empty repo.
Copy your code from the project that you completed during the Admissions Trial: - If you do not have the code from the Admissions Trial ask someone from your Standup Team to share their code with you. In that case, remember to add an attribution note and acknowledgment in your README file.
Create a new branch for your copied code.
Open a pull request

### CODE REVIEW

**Comment by Kaneez98Fatima** [comment]
Hi @Kaneez98Fatima,

Very nice project. I love that you have remembered to add node_modules to the .gitignore 👏🏿 . However, there are a few changes to be made kindly check below ⬇️

Required Changes ♻️
 There should be a test.md file in the .gitIgnore file. This is a hard requirement for this project. Kindly create a test.md file and make sure you add it to the .gitIgnore. You can use this article as a reference.
 When writing your commit messages, it will be good to always use the right tense and also remember to start your statement with a capital letter. This will make your project even more professional. For example, instead of Readme file updated it should be an Update readme file
For more check the comments under the review.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Status: Invalid ❗ 

### Hi @Kaneez98Fatima 👋🏾 , 

 **While you made a great effort in this project, unfortunately, I cannot proceed to review your code.**

### Invalid Code Review Request

I have to mark this submission as invalid because you have not implemented the project requirements which is to perform a code review on your code. Remember to add a minimum of 2 comments in the review as well as a summary of that review. Endeavour to use GitHub features for the reviews.



#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by teekaytech — CHANGES_REQUESTED** [review]
Hi @Kaneez98Fatima, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a proper setup of GitHub flow. Although you attempted to set up the GitHub Actions Linters, your setup was not done in the right way. Carefully follow [these instructions](https://github.com/microverseinc/linters-config/tree/master/html-css) to properly set up your linters.

Also, you need to understand the task at hand here. As mentioned by the last reviewer, this project is about **you performing a code review on one of your projects and applying HTML & CSS best practices introduced by the [How to give a code review?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) lesson.** 
- This means that after building the project (as you've done), you'll perform a review based on the link above. 
- After reviewing your own code, you'll submit a code review request, using the PR link of your review. 
- If you're having any difficulty understanding the project or you need any help, kindly [schedule a one-on-one meeting with a full-time Code Reviewer](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-).

--------

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Addimission_trial_code.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
```
Comment (Kaneez98Fatima) [inline Addimission_trial_code.html:10]: I would suggest you remove the blank lines or white spaces to create a clean code. :smile:

File: Addimission_trial_code.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  <body>
+      <h1 id="title">Survey Form</h1>
+  <br>
+      <p id="description">This is a survey form</p>
+<br>
```
Comment (Kaneez98Fatima) [inline Addimission_trial_code.html:15]: **Required changes** 
I  would suggest you fix the indentation with the br tag in this way you didn't face any difficulty during the checking.
Good Luck :+1:

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
-# empty
\ No newline at end of file
+# Code Review Trial
```
Comment (Temmarie) [inline README.md:1]: - [ ] You've done a good job on this review, following the requirements for this project. However, you need to make sure the comment under a line or block of code fits the context the comment is about. A comment like this should be placed in the review summary.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Kaneez98Fatima      ! :wave: :wave:

#### Good job so far! You have done really well in implementing the requirements.  
- You have used GitHub inline comment ✔️ 
- You have added a summary at the end of the review ✔️ 
- You made suggestions using polite terms and giving reasons for them ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!
- - -

### **Required changes** ♻️
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- I suggest going through [this](https://www.markdownguide.org/cheat-sheet/) article to know how to use Markdowns.
- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comment under one single summary like this current review. :wink:

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by Kaneez98Fatima — COMMENTED** [review]
### Changes Required :recycle: 
Hello @Kaneez98Fatima  :wave:
You done a good job ..your project is very good you work hard a lot but there are some changes which I suggest you to improving your code skills :hugs: 

Kindly check the comments below and submit the changes

When writing your commit messages, it will be good to always use the right tense and also remember to start your statement with a capital letter. 

This will make your project even more professional.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  margin: 10px auto;
+  color: rgb(66, 66, 128);
+}
+
+#description {
+  text-align: center;
+  font-size: 25px;
+  color: #fff;
+}
+
```
Comment (jubaan) [inline styles.css:24]: - [ ] Great job posting a message asking for "changes"!, Remember that this project is about your ability to use GitHub review tools to review/comment your code or other developers in a way that is almost self explainatory. 

  If you write a message asking to the developer to fix some issues in the code he is working on, and point out that the requested changes will follow the message, we should always make sure those notes are below our initial message in `inline comments` like this one.

  You're almost there, let's give it another chance.

**Review by jubaan — CHANGES_REQUESTED** [review]
Hi @Kaneez98Fatima , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kaneez98Fatima — COMMENTED** [review]
### Changes Required :recycle: 

Hello @Kaneez98Fatima wave
You done a good job ..your project is very good you work hard a lot but there are some changes which I suggest you to improving your code skills hugs

Kindly check the comments below and submit the changes

I would suggest you fix the indentation with the br tag in this way you didn't face any difficulty during the checking.
Good Luck +1
I would suggest you remove the blank lines or white spaces to create a clean code. smile

### Optional Changes :+1: 

When writing your commit messages, it will be good to always use the right tense and also remember to start your statement with a capital letter.

This will make your project even more professional :1st_place_medal:

**Review by jubaan — APPROVED** [review]
Hi @Kaneez98Fatima , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**